Pd 331 rbac (#697)
portworx/pxdocs: content/portworx-install-with-kubernetes/storage-operations/rbac-stork/_index.md

---
title: "Configure role-based access control for Stork"
linkTitle: "Configure RBAC for Stork"
keywords: cloud, backup, restore, snapshot, DR, migration, kubemotion, stork, rbac, role, clusterrole
description: Learn how to configure role-based access control to allow your users to perform Stork operations.
noicon: true
weight: 11
---

Cluster administrators can allow certain users to perform Stork operations by creating and assigning custom roles for their users. Under the configuration described below, a Cluster administrator defines permissions in a user role which allows the user to perform Stork operations.

## Modify a role

> 1. As a Kubernetes cluster administrator, create a new Kubernetes cluster role, or find an existing role you wish to modify.

Reviewer: ```suggestion
1. As a Kubernetes cluster administrator, create a new Kubernetes role, or find an existing one you wish to modify.
```

Reviewer: `Role` is an object type, so maybe you want to highlight that?

---
title: "Configure role-based access control for Stork"

> linkTitle: "Configure access control for Stork"

Reviewer: ```suggestion
linkTitle: "Configure role-based access control for Stork"
```

> keywords: cloud, backup, restore, snapshot, DR, migration, kubemotion

Reviewer: ```suggestion
keywords: cloud, backup, restore, snapshot, DR, migration, kubemotion, stork, rbac, role, clusterrole
```

description: Learn how to configure role-based access control to allow your users to perform Stork operations.
noicon: true
weight: 11
---

Cluster administrators can allow certain users to perform Stork operations by creating and assigning custom roles for their users. Under the configuration described below, a Cluster administrator defines permissions in a user role which allows the user to perform Stork operations.

## Prerequisites

> * A running Portworx cluster on Kubernetes with role-based access control

Reviewer: Don't need to mention this. The role can be added even if there is no PX or stork

> ## Modify a cluster role

Reviewer: ```suggestion
## Modify a Kubernetes role
```
or 
```suggestion
## Modify a role
```

Role and ClusterRole are 2 different types of objects

1. As a Kubernetes cluster administrator, create a new Kubernetes cluster role, or find an existing role you wish to modify.

> 2. Modify the role, adding the Stork rules to it:

Reviewer: ```suggestion
2. Modify the role, adding permissions for Stork resources to it:
```

>         apiVersion: rbac.authorization.k8s.io/v1

Reviewer: This should be in a code box?

Author: In this case, no. The `-` character doesn't work with the traditional code box syntax, this level of indentation is an alternative way of creating a code box.
